test/ts/unit/collateralizer/scenarios/successful_return.ts

```diff
@@ -0,0 +1,62 @@
+import { ReturnCollateralScenario } from "../runners/";
+
+// external
+import { BigNumber } from "bignumber.js";
+import * as moment from "moment";
+
+// utils
+import * as Units from "../../../test_utils/units";
+
+export const SUCCESSFUL_RETURN_SCENARIOS: ReturnCollateralScenario[] = [
+    {
+        description: "Debt's term has lapsed and debt has been repaid",
+        collateralAmount: Units.ether(3),
+        gracePeriodInDays: new BigNumber(7),
+        valueRepaidToDate: Units.ether(1),
+        expectedRepaymentValueSchedule: [
+            {
+                timestamp: moment()
+                    .subtract(14, "days")
+                    .unix(),
+                expectedRepaymentValue: Units.ether(0.5),
+            },
+            {
+                timestamp: moment()
+                    .subtract(7, "days")
+                    .unix(),
+                expectedRepaymentValue: Units.ether(1),
+            },
+        ],
+        termEndTimestamp: 0,
```

Comment:
This very well may be an issue I introduced originally, but I don't understand why this `termEndTimestamp` is 0.  Seems like it should be a timestamp after the last timestamp in `expectedRepaymentValueSchedule` and before now.

Reply:
It's actually just mocked that way in MockCollateralizedTermsContract!